src/commands/publish/index.ts

```diff
@@ -0,0 +1,21 @@
+import { Command, Option } from 'commander';
+import { publishHandler } from './publishHandler';
+
+export const publish = new Command('publish');
+
+publish
+  .description('publish a Plugin to a PluginRepo')
```

Comment:
important to keep consistent across capitalization here. either all lower cased or all upper cased. let's go for capitalizing all of them.